modules/migration-upgrading-migration-tool-3.adoc

```diff
@@ -70,13 +70,20 @@ $ oc -o yaml -n openshift-migration get deployment/migration-operator | grep ima
 $ sudo podman cp $(sudo podman create registry.redhat.io/rhmtc/openshift-migration-rhel7-operator:v{mtc-version-z}):/controller-3.yml ./
 ----
 
-. Create the `MigrationController` object:
+. Create the `migration-controller` object:
 +
 [source,terminal]
 ----
 $ oc create -f controller-3.yml
 ----
 
+. If your {product-title} version is 3.10 or earlier, set the security context constraint of the `migration-controller` to `anyuid` to enable direct image migration and direct volume migration:
```

Comment:
I think we should add that the migration-controller needs to be installed and that you are adding the security context constraint to the migration-controller service account?

Reply:
The controller is created in the previous step. I can try to make the SA aspect clearer.